src/zenml/utils/package_utils.py

```diff
@@ -0,0 +1,27 @@
+#  Copyright (c) ZenML GmbH 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from importlib.util import find_spec
+
+
+def is_installed(module_name: str) -> bool:
```

Comment:
I think this might belong in `source_utils` rather than creating another file tbh

Reply:
Yes, I agree. Think is, when I add it into `source_utils` I get a circular import since `source_utils` imports `Environment`. Do you think it's better to keep it in `package_utils` or to move the whole function just as a helper in `Environment` that isn't part of our utils?

Reply:
my suggestion would be to simply discard this function entirely. When the signature of the function is longer than the body, it probably doesn't warrant a dedicated function.